### DISCUSSION TITLE
Colors use variables and are called via var

### DISCUSSION THREAD

**Discussion by amihhs** [unocss/unocss#2816] [Ideas]
Whether to consider the use of variables for color and then call it through var
eg.
![image](https://github.com/unocss/unocss/assets/59364234/397e83ed-8aa8-49d5-b031-c1cd782dbe56)

This makes it easy to inherit it elsewhere
![image](https://github.com/unocss/unocss/assets/59364234/c126eac7-1cf0-4910-baac-222e81e56192)

@antfu @zyyv

**Comment by zyyv** [comment]
I think we should align with tailwind and windi. Alternatively you can easily achieve this using `postprocess`  
[Playground](https://unocss.dev/play/?html=DwEwlgbgBA7gTgQwA5IKZwHwCgpVJKAF1QA9CBaOVEDAVQDsB7AYQGVXgB6cCbXfaDHIBGAAxQAFiPEAjAObliCADZRO2Lj2xA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbhwqAC2gYYQ4qAC5Ebx84AHcoKzAwVChwjWiAWgBGAAYsgDdouD9MnNy-ZVtTOGAYVBAqNOFULBqoOAArNipOHgamlo0SSIVBn34pMLgAbUjRyVpZCHcAShGosdoGJlYOLm5l1dmBGBExTxmoqmErCvCMgDoAJgOo4WQsFL9GMDCAeh-qEB3fw-AbnBQrSIAXVWkC6FAgjT04XcO1sSzgAF4AHwRKJwURYLpwKBEFhWABKAFFMXAfiSyQAddzuBnIADUJFZHK5K3ckwAektIQAqJYMpY-c6ou59ZjUO7oaCUqzCPzM4Do7G4vH4sTwXLXNioGnASYZSHnHxcODuGDcRIQdBwA22I2YjEYzRdZhYIgaOAAMgDzsNqDuICsQTV9IplPRAEJPVg7GjtTrdYT4BGozSXUbw5HVe4Y1SlpaoqbzbnQ3dKGBbCrUMXSbGyBoY%2B4DVB3Gk0toGhBbNAVgASB5LDRl9M%2BaWy4Dy7T%2BLgwPkaXv90RDqADODZ1VmyGQqc6hRg4-DAzggxAA&css=Q&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g)

**Comment by antfu** [comment]
For the wind preset, we aim to align with Tailwind. It's definitely possible for you to write your own presets tho.

**Comment by amihhs** [comment]
postprocess is a great solution, it only requires a little extra configuration. Thanks
